contracts/Minter.sol

```diff
@@ -0,0 +1,164 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.11;
+
+interface IMintable {
+  function mint(address to, uint256 amount) external;
+  function decimals() external returns (uint256);
+}
+
+/// @title A Minter contract for Splinterlands
+/// @author Splinterlands Team (@fbslo)
+
+contract Minter {
+  /// @notice Address that can change pools
+  address public admin;
+  /// @notice Address of the token to mint
+  address public token;
+  /// @notice Block number when mint() was last called
+  uint256 public lastMintBlock;
+  /// @notice Total number of tokens already minted
+  uint256 public totalMinted;
+  /// @notice Maximum number of tokens minted, 3B (with 18 decimal places)
+  uint256 public cap = 3000000000000000000000000000;
+  /// @notice Maximum number of pools
+  uint256 public poolsCap = 100;
```

Comment:
should be declared as `constant`

Reply:
resolved in https://github.com/steem-monsters/sps-minter/commit/08b460f427d034b902deaf79c22a70576d41d129